packages/framework/dailyTableUpdates/table_Automation.py

```diff
@@ -51,39 +57,47 @@ def main():
         last_Friday = today + relativedelta(weekday=FR(ONE_OFF))
     
     except:
-        print("View needed dependencies above\n\n")
-        
+        print("Veiw needed dependencies above\n\n")
+
     try:
         number_of_pr_merged = stat_container[1]
         number_of_failed_pr = stat_container[2]
-        number_of_waiting_pr = stat_container[3]
-        number_open_pr = stat_container[4]
-        number_of_successful_mm = stat_container[6]
-        jira_ticket_number = stat_container[7]
+        number_wip_prs = stat_container[3]
```

Comment:
Setting up an ArgumentParser() is probably more robust in the long term so that you're not relying on a bunch of order-dependent input parameters (and has the added bonus of generating a --help usage message automatically).

Reply:
Hi, I have updated the script to use a ArgumentParser.